### ISSUE TITLE
[Feature Request] Native Fingerprint Support in Cinnamon

### ISSUE THREAD

**Issue by forever-and-a-day** [linuxmint/cinnamon#10234]
KDE Plasma and GNOME have already adopted fingerprint support right into their desktop environments. Many have manually added fingerprint GUIs and CLIs to cinnamon-based systems (such as Linux Mint), but these tools often break, have setup difficulty, and poor system integration. As a distribution aiming to provide a traditional, yet **modern** experience, fingerprint authentication should be a high priority, from integration into Cinnamon's settings, system setup in Mint, and authentication prompts. This could even extend into terminal authentication, if possible and viable. I think that in general, when users want to switch to linux, find mint or another distro with cinnamon, and realize that a major feature of their hardware (a lot of laptops included) doesn't function, it will push a lot of users away.

**Comment by glgkgjglglkglk** [comment]
Yes. I support this feature request too. I have two notebooks wtih fingerprint reader

**Comment by forever-and-a-day** [comment]
That's okay. The point of this inclusion is to bring feature parity hardware-wise to the likes of windows and macos. People have hardware, and it should work with their OS.

**Comment by forever-and-a-day** [comment]
Any updates on this? Haven't had a chance to try out LM Zara yet

**Comment by mtwebster** [comment]
This is provided by the fingwit package now as of Mint 22.2 and LMDE7:
https://blog.linuxmint.com/?p=4850
